crates/iroha_executor/src/permission.rs

```diff
@@ -1084,3 +1085,29 @@ pub(crate) fn roles_permissions(host: &Iroha) -> impl Iterator<Item = (RoleId, P
                 .map(move |permission| (role.id().clone(), permission))
         })
 }
+
+/// Revoked all permissions satisfied given [condition].
+///
+/// Note: you must manually call `deny!` if this function returns error.
```

Comment:
Yeah, this is not a comment on this PR, but why do we have executor visit instructions and mutate its verdict, and why not propagate errors?

Reply:
because we didn't want to make assumptions on what `visit_xxx` does